test_simple_graph.py

```diff
@@ -1,5 +1,7 @@
 import pytest
 from simple_graph import Graph
+
+
```

Comment:
You should also run appropriate tests for a graph that has an "orphan" node - one that is not connected by an edge to anything else.